src/components/InteractiveTutorial/TutorialSubpages.jsx

```diff
@@ -13,6 +14,8 @@ export const tutorialIndexPage = Index;
 const tutorialSubPages = [
   ['quickstart', Quickstart],
   ['filtering-clauses', FilteringClauses],
+  ['multivectors', Multivectors],
+  ['multitenancy', Multitenancy]
```

Comment:
```suggestion
  ['multitenancy', Multitenancy],
```
To fix the lint failure.